### ISSUE TITLE
rows.first() gives ValueError: RecordCollection contains too many rows

### ISSUE THREAD

**Issue by leafonsword** [kennethreitz/records#122]
> records: 0.5.2
   Python: 3.6
   MySQL driver: mysqlclient (1.3.12)

```
>>> rows = db.query('select * from tmpRDSBak_20180102170341SMS_Unsub_Month_BG;') 
>>> rows.first()
Traceback (most recent call last):
  File "<input>", line 1, in <module>
    rows.first()
  File "/usr/local/miniconda2/envs/py3_env/lib/python3.6/site-packages/records.py", line 222, in first
    raise ValueError('RecordCollection contains too many rows.')
ValueError: RecordCollection contains too many rows.
```
**But this table just contains 13 rows**
```
>>> rows = db.query('select count(*) from tmpRDSBak_20180102170341SMS_Unsub_Month_BG;')
>>> rows.first()
<Record {"count(*)": 13}>  
```

**Comment by kennell** [comment]
I just stumbled across the same error. Looking at the relevant code part, it seems that raising this exception is the intended behaviour of `first()` whenever there are >1 rows. See: https://github.com/kennethreitz/records/blob/master/records.py#L205

I feel like this is a bad naming choice. Given the wording, i would expect `first()` to give me the first element, regardless of how many rows have been returned by ther query. Maybe introduce something like `one()` that returns exactly one row or raises a exception if >1 rows are retreived.

**Comment by Tethik** [comment]
Coming from a .NET background this weirded me out a bit too. I'm used to [`first()` returning the first item](https://msdn.microsoft.com/en-us/library/bb291976(v=vs.110).aspx) of a sequence of any length, while [`single()` ensures that there is also only one item](https://msdn.microsoft.com/en-us/library/bb155325(v=vs.110).aspx).

Sqlalchemy has [`first()` return the first row](http://docs.sqlalchemy.org/en/latest/orm/query.html#sqlalchemy.orm.query.Query.one) (although it also does a limit 1) and uses [`one()`](http://docs.sqlalchemy.org/en/latest/orm/query.html#sqlalchemy.orm.query.Query.first) to return and enforce.

I think I'd go for the sqlalchemy naming convention (first/one), since thats more closely python related :snake:

**Comment by vlcinsky** [comment]
Resolved by PR #127

**Comment by kennell** [comment]
Thanks guys!
